### PR TITLE
Classical damage calculator

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1389231.
The tests are green: https://ci.openquake.org/job/zdevel_oq-engine/962

### CODE REVIEW

File: openquake/engine/db/schema/upgrades/0013-classical_damage.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
+-- Damage curve
+CREATE TABLE riskr.damage (
+    id SERIAL PRIMARY KEY,
+    risk_calculation_id INTEGER NOT NULL REFERENCES uiapi.oq_job,
+    output_id INTEGER NOT NULL REFERENCES uiapi.output,
+    hazard_output_id INTEGER NULL REFERENCES uiapi.output,
```
Comment (ptormene) [inline openquake/engine/db/schema/upgrades/0013-classical_damage.sql]: Please check if this is fine, or if it should be `NOT NULL` or without `NULL`

File: openquake/engine/db/schema/upgrades/0013-classical_damage.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+COMMENT ON TABLE riskr.damage IS 'Holds the parameters common to a set of damage curves.';
+COMMENT ON COLUMN riskr.damage.output_id IS 'The foreign key to the output record that represents the corresponding damage curve.';
+
+-- Damage curve data
+CREATE TABLE riskr.damage_data (
+    id SERIAL PRIMARY KEY,
+    damage_id INTEGER NOT NULL REFERENCES riskr.damage,
+    dmg_state_id INTEGER NOT NULL REFERENCES riskr.dmg_state,
+    exposure_data_id INTEGER NOT NULL REFERENCES riski.exposure_data,
```
Comment (ptormene) [inline openquake/engine/db/schema/upgrades/0013-classical_damage.sql]: is `riski` correct, or maybe `riskr`?

File: openquake/engine/db/schema/upgrades/0013-classical_damage.sql (hunk 1 above)
Comment (micheles) [inline openquake/engine/db/schema/upgrades/0013-classical_damage.sql]: NULL is fine. This happens for statistics damage outputs which are not associated to any realization.

File: openquake/engine/db/schema/upgrades/0013-classical_damage.sql (hunk 2 above)
Comment (micheles) [inline openquake/engine/db/schema/upgrades/0013-classical_damage.sql]: riski is correct

**Comment by ptormene** [comment]
LGTM

**Pull request merged**